setup.py

```diff
@@ -52,7 +52,8 @@ def run_tests(self):
     'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
     'pytest>=2.6.1',
-    'coverage<4.0a1'
+    'coverage<4.0a1',
+    'flask-classy>=0.6.10'
```

Comment:
do you need it also in `extras_require = { ..., 'classy': ['flask-classy>=0.6.10]}`?